cfgov/unprocessed/css/molecules/expandable.less

```diff
@@ -0,0 +1,272 @@
+                                   m-expandable_cue-open">
+                          <span class="m-expandable_cue-label">Show</span>
+                          <span class="cf-icon cf-icon-plus-round"></span>
+                      </span>
+                      <span class="m-expandable_cue
+                                   m-expandable_cue-close u-hidden">
+                          <span class="m-expandable_cue-label">Hide</span>
+                          <span class="cf-icon cf-icon-minus-round"></span>
+                      </span>
+                  </span>
+              </div>
+          </button>
+          <div class="m-expandable_content" aria-expanded="false">
+          </div>
+      </div>
+    codenotes:
+      - |
+        Pattern structure
+        -----------------
+        .m-expandable
+          button.m-expandable_target
+            .m-expandable_header
+              .m-expandable_header-left.m-expandable_label
+              .m-expandable_header-right.m-expandable_link
+                .m-expandable_cue-open
+                  .m-expandable_cue-label
+                .m-expandable_cue-close
+                  .m-expandable_cue-label
+          .m-expandable_content
+    notes:
+      - "Use the button tag for the .expandable_target element to allow for
+         keyboard access."
+      - "The 'Show' and 'Hide' messages can be customized directly in the HTML
+         by editing the contents of .m-expandable_cue-label."
+      - "The aria-pressed attribute on .m-expandable_target gets automatically
+         added and updated by JavaScript."
+      - "The aria-expanded attribute on .m-expandable_content gets automatically
+         updated by JavaScript."
+  - name: .m-expandable__expanded (modifier)
+    codenotes:
+      - |
+        .m-expandable__expanded
+    notes:
+      - "Sometimes you may want the expandable to be open by default. This
+         is as easy as adding the .m-expandable__expanded modifier to the
+         .m-expandable block."
+  tags:
+  - cf-molecules
+*/
+.m-expandable {
+    .webfont-regular();
+    padding-bottom: unit( ( @grid_gutter-width * 2 ) / @base-font-size-px, em );
+
+    &_target {
+        // Override button default styles.
+        display: block;
+        background: none;
+        border: none;
+        width: 100%;
+        position: relative;
+        cursor: pointer;
+
+        border-top: 1px solid @gray-50;
+        background: @white;
+    }
+
+    &_target,
+    &_content-animated {
+        padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    }
+
+    &_content-animated {
+        position: absolute;
+        bottom: 0;
+        padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+    }
+
+    &_content {
+        height: 0;
+        border-bottom: 1px solid @gray-50;
+        overflow: hidden;
+        position: relative;
+        background: @white;
+
+        &-animated {
+            position: relative;
```

Comment:
This is overwriting L:132. Also, all styles on `_content-animated` should be in one place.